kernels/volk/volk_32u_popcntpuppet_32u.h

```diff
@@ -18,10 +18,8 @@ static inline void volk_32u_popcntpuppet_32u_generic(uint32_t* outVector,
                                                      const uint32_t* inVector,
                                                      unsigned int num_points)
 {
-    unsigned int ii;
-    for (ii = 0; ii < num_points; ++ii) {
-        volk_32u_popcnt_generic(outVector + ii, *(inVector + ii));
-    }
+    for (size_t i = 0; i < num_points; ++i)
+        volk_32u_popcnt_generic(outVector + i, inVector[i]);
```

Comment:
I really prefer braces around one-liners. This bit me before and I usually have a hard time to read it.